src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.StringSplitter.cs

```diff
@@ -10,6 +9,8 @@
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.SplitStringLiteral
 {
+    using Microsoft.CodeAnalysis.Indentation;
```

Comment:
move to where other using is?

Reply:
can't.  if we move it, then diffrent symbols will be found in Microsoft.CodeAnalysis.Editor.

Reply:
so, we have a plan to move at some point then? or it has to be here until typescript/F# is moving to new API as well?

Reply:
Yes, this is tracked with: https://github.com/dotnet/roslyn/issues/35872